src/services/IbexHelper/IbexHelper.ts

```diff
@@ -0,0 +1,56 @@
+import axios, { isAxiosError, AxiosError, AxiosRequestConfig } from "axios"
+
+// import IBEX_TOKEN from .env file
+const IBEX_TOKEN = process.env.IBEX_TOKEN
+const IBexPluginBaseURL = "http://api-development.flashapp.me:8760/api/v1"
+const token = IBEX_TOKEN
+
+const defaultHeaders: AxiosRequestConfig["headers"] = {
+  "Authorization": token,
+  "Content-Type": "application/json",
+}
+
+export async function requestIBexPlugin(
+  methodType: string,
+  endpoint: string,
+  params: object,
+  body: object,
+) {
+  const result = {
+    status: 400,
+    data: null,
+    error: "",
+  }
+
+  try {
+    let response = { status: 200, data: null, error: "" }
+    const requestOptions: AxiosRequestConfig = {
+      headers: defaultHeaders,
+    }
+
+    if (methodType === "GET") {
```

Comment:
maybe a switch statement might make more sense here but this works aswell

Reply:
changed to switch statement